charts/tekton-apps/templates/wordpress/argocd_applications.yaml

```diff
@@ -230,8 +230,13 @@ spec:
           {{- end }}
 
         persistence:
+        {{- if ($component.wordpress).persistence }}
+        {{- toYaml $component.wordpress.persistence | nindent 10 }}
+        {{- else }}
+        {{- /* For compatibility with versions priorior to 0.1.27 */}}
           enabled: true
           storageClass: gp2
```

Comment:
@populov what happens if we don't have the `else` block?

Reply:
If we remove compatibility block, then we will have to set at least `storageClass: gp2` for every wordpress project, because bitnami chart has default `storageClass: ""`

Reply:
@populov then instead of this block - pls go through tekton-apps and adjust values.yaml in rocks cluster properly to have a proper block. That would be a right way of doing it.

Reply:
@dmitry-mightydevops Maybe let's leave this `default` else section as it is? So the way I see it - it is not for compatibility, it is a default `persistence` configuration, when it is not set.  So we avoid adding the same defaults for all our WP projects in rocks. Why it is bad to have defaults?

Reply:
@kseniyashaydurova explicit is better than implicit. So I'd like to have full declarations in the tekton-apps in this PR for all wordpress components
https://github.com/saritasa-nest/saritasa-rocks-kubernetes-aws/pull/105

Then it's better than the implicit solution through the else block for older wordpress components we have in the tekton-apps rocks today.

:) 
![image](https://user-images.githubusercontent.com/70822710/207160746-f3e7a3ea-2018-4dca-9d6d-ec9960d7eb8c.png)